src/app/formatxml.py

```diff
@@ -13,6 +13,7 @@
 
 INDENT = 2
 LINE_LENGTH = 80
+IS_EXCEPTION = False
```

Comment:
Having a global variable can affect other users as well since the global value is changed. The application won't have separate instances of `app` for every user.